tsstats/tests/test_template.py

```diff
@@ -35,7 +35,8 @@ def test_debug(output):
     logger.setLevel(logging.INFO)
     soup = BeautifulSoup(open(output_path), 'html.parser')
     # check debug-label presence
-    assert soup.find('nav').find(id='main-nav').find('span').text == 'debug mode'
+    debug_element = soup.find('nav').find('div', id='main-nav').find('span')
```

Comment:
I didn't think there was a neat place to split the line using a backslash. Any positioning would have made the code far less readable than just keeping it all on one line IMO.